### ISSUE TITLE
Output model generates noisy sounds, robotic voices

### ISSUE THREAD

**Issue by thucth-qt** [tuanh123789/Train_Hifigan_XTTS#15]
Hi @tuanh123789  after training for 3 days. My output model generates noisy sounds, robotic voices.
This is my dataset specs:
- over 400 hours of speech (Vietnamese total)
- clear, high quality dataset.
- diverse accent, gender,.. collected from youtube
- I finetune HifiGAN from ckpt: https://huggingface.co/capleaf/viXTTS
- I have trained for 155127 epochs and here is the output (drive [file attached](https://drive.google.com/drive/folders/106bhtagBZ21ImC_d2jh23KjK3YtTll2U?usp=sharing))
I appreciate so much that if you can support me this issue.

**Comment by tuanh123789** [comment]
Hi, what batch size, how many GPU do you use to training, with 400 hours i think you should leave the training countinue.
And note that this version work better with single speakers, just chose one target to finetune for best result. The multi-speaker train version contains a loss function that I haven't added yet

**Comment by xiaoyangnihao** [comment]
> Hi, what batch size, how many GPU do you use to training, with 400 hours i think you should leave the training countinue. And note that this version work better with single speakers, just chose one target to finetune for best result. The multi-speaker train version contains a loss function that I haven't added yet

@tuanh123789 hi，is multi-speaker version is ready for now?
